docs/src/css/components/_dialog.scss

```
@@ -1,42 +1,125 @@
 dialog {
-  padding: var(--ix-space-md);
-  border-radius: var(--ix-radius);
+  padding: var(--ix-space-lg);
+
+  border-radius: var(--ix-radius-lg);
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)